doc/Makefile

```diff
@@ -196,19 +195,12 @@ GZFILES_1_BUILD = $(addprefix $(MANPAGES_BUILDDIR)/, $(notdir $(GZFILES_1)))
 
 GZFILES_3_BUILD += $(addprefix $(MANPAGES_BUILDDIR)/, $(GZFILES_3_DUMMY))
 
-DOXYGEN_HTMLDIR=cpp_html
-
 MANPAGES_DESTDIR_7 = $(DESTDIR)$(man7dir)
 MANPAGES_DESTDIR_5 = $(DESTDIR)$(man5dir)
 MANPAGES_DESTDIR_3 = $(DESTDIR)$(man3dir)
 MANPAGES_DESTDIR_1 = $(DESTDIR)$(man1dir)
 
 DOCS_DESTDIR = $(DESTDIR)$(docdir)
-CPP_DOC_DIR ?= libpmemobj++-dev
-CPP_DOCS_DESTDIR = $(DOCS_DESTDIR)/$(CPP_DOC_DIR)
-
-
-all: md2man $(TXTFILES) doxygen_docs | $(MANPAGES_BUILDDIR)
```

Comment:
I think only doxygen_docs should be removed

Reply:
Done.